transport-native-epoll/src/main/java/io/netty/channel/epoll/Native.java

```diff
@@ -43,11 +43,15 @@
 public final class Native {
 
     static {
-        String name = SystemPropertyUtil.get("os.name").toLowerCase(Locale.UK).trim();
-        if (!name.startsWith("linux")) {
-            throw new IllegalStateException("Only supported on Linux");
-        }
-        NativeLibraryLoader.load("netty-transport-native-epoll", PlatformDependent.getClassLoader(Native.class));
+       loadLibrary();
+    }
+
+    public static void loadLibrary() {
+       String name = SystemPropertyUtil.get("os.name").toLowerCase(Locale.UK).trim();
```

Comment:
@normanmaurer - I'm more asking about the specific choice of `Locale.UK`.  Does it have to be this specific Locale? What if this is not available, and another that may work is available (for example the default)?